From 5da394897e9ed188310ca838f5f9112687b7bbe9 Mon Sep 17 00:00:00 2001
From: "Allan W. Nielsen" <allan.nielsen@microchip.com>
Date: Tue, 12 Sep 2023 20:07:07 +0200
Subject: [PATCH] EF: Bump cmake version

Review-record: https://bitbucket.microchip.com/projects/UNGE/repos/sw-lmstax-labs/pull-requests/679

Upstream: https://github.com/microchip-ung/easyframes/commit/5da394897e9ed188310ca838f5f9112687b7bbe9

[Github diff munged slightly to avoid including the UTF-8 BOM]
Signed-off-by: Florian Larysch <fl@n621.de>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cf7bad1..e489537 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,4 +2,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.20)
 
 option(TEST_ENABLE "Enable tests" off)
 
